3-plotting/manual.ipynb

```diff
@@ -673,6 +673,96 @@
     "plt.xscale(\"log\")\n",
     "plt.yscale(\"log\")"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Customizing Matplotlib\n",
+    "\n",
+    "There are many ways of changing the style of a figure in Matplotlib.\n",
+    "Here we will go through a few.\n",
+    "\n",
+    "#### 1) Directly\n",
+    "As has been demonstrated above, to change the properties of one individual element one can pass the setting directly to that element. Examples include\n",
+    "```python\n",
+    "plt.plot(x, y, color=\"red\", linestyle=\"-.\")\n",
+    "```\n",
+    "and\n",
+    "```python\n",
+    "plt.xlabel(\"Temperature [$\\degree$C]\", fontsize=16)\n",
+    "```\n",
+    "\n",
+    "#### 2) Style sheets\n",
+    "In the presentation, we used a style sheet\n",
+    "```python\n",
+    "plt.style.use(\n",
+    "    \"https://lund-observatory-teaching.github.io/lundpython/3-plotting/presentation.mplstyle\"\n",
+    ")\n",
+    "```\n",
+    "A style sheet is a document containing settings for Matplotlib. By loading a style sheet with the command shown above the settings in the style sheet will be applied to all subsequent plots.\n",
+    "You can use the one shown here, or make your own. \n",
+    "Matplotlib also provides a list of ready-made style sheets [here](https://matplotlib.org/stable/gallery/style_sheets/style_sheets_reference.html) for anyone to use.\n",
```

Comment:
Pointing out hyperlinks with words like "here" or "this" is not a good practice.
```suggestion
    "Matplotlib also provides [a list of ready-made style sheets](https://matplotlib.org/stable/gallery/style_sheets/style_sheets_reference.html) for anyone to use.\n",
```